pnpm-lock.yaml

```diff
@@ -1453,8 +1453,8 @@ packages:
   chessground@9.0.4:
     resolution: {integrity: sha512-KeZd/kcXSKQOGO6LOKu/3bAyVSMh7hgPPharMF15HMfNlHfcBpIVZIDAQRO2PNaNMzPWl0LKc05uFHt0D+lZGA==}
```

Comment:
should we remove this?

Reply:
We don't manually edit this file or lock files in general; they are generated. Not sure why this older version is here, maybe a different page is using in older chessground.